src/views/preview/comment/comment.jsx

```diff
@@ -7,6 +7,8 @@ const FlexRow = require('../../../components/flex-row/flex-row.jsx');
 const Avatar = require('../../../components/avatar/avatar.jsx');
 const FormattedRelative = require('react-intl').FormattedRelative;
 const ComposeComment = require('./compose-comment.jsx');
+const DeleteCommentModal = require('../../../components/modal/comments/delete-comment.jsx');
```

Comment:
Why does comment.jsx live in the views/preview dir instead of a components/comment dir? Seems like that would make more sense

Reply:
Yeah, that was a future improvement from that first comment PR. That which lives in components/comment is actually used for a different thing. I was going to do some of those cleanup things in a separate PR.